scripts/packages/pyheron/README.txt

```diff
@@ -0,0 +1,2 @@
+PyHeron is the python API for Heron, the distributed streaming engine. The API is
+based on the Storm python API provided by Parsely.
```

Comment:
This is required for Python packaging - so that we can submit to pypi. When we package it, the packager specifies that the readme file should either README or README.txt or README.rst (not README.md)